platform/ios/docs/guides/UserInteractions.md

```diff
@@ -0,0 +1,47 @@
+# User Interactions in the Mapbox iOS SDK
+
+The Mapbox iOS SDK provides a set of built-in gesture recognizers to developers. These interactions can be customized to fit your use case. You see what gesture recognizers are on your `MGLMapView` by accessing the `gestureRecognizers` property on your map.
+
+## Configuring user interaction
+
+There are several properties on a `MGLMapView` that are built-in ways to enable or disable a set of gesture recognizers. Boolean values are set to `true` by default.
+
+- `zoomEnabled` - Allows the user to zoom in and out by pinching two fingers, double tapping, or using "quick zoom". Accepts Boolean values.
```

Comment:
Avoid the term “quick zoom” in documentation, since this term is little-known among iOS developers. Instead, say “or double-tapping then dragging vertically”.